editor/app/src/main/java/ca/ubc/eml/soiltopargraphy/editor/MainActivity.kt

```diff
@@ -4,7 +4,13 @@ import android.support.v7.app.AppCompatActivity
 import android.os.Bundle
 import ca.ubc.eml.soiltopargraphy.editor.ui.main.MainFragment
 
-class MainActivity : AppCompatActivity() {
+
```

Comment:
❓

Reply:
I did not actually change anything here, just accidentally deleted it and then re-added it.